guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -5,13 +5,24 @@
 If the custom code is executed before and/or after the provisioning process, use custom provisioning snippets to avoid recreating cloned templates.
 For more information about configuring custom provisioning snippets, see {ProvisioningDocURL}Creating_Custom_Provisioning_Snippets_provisioning[Creating Custom Provisioning Snippets] in _{ProvisioningDocTitle}_.
 
+* Delete the `/var/lib/pgsql/data-old/` directory:
++
+[options="nowrap" subs="attributes"]
+----
+# rm -r /var/lib/pgsql/data-old/
+----
++
+{ProductName} creates this directory for backup purposes during the upgrade.
+If the upgrade finishes successfully, the data in this directory can no longer be used.
+
 ifdef::katello,orcharhino,satellite[]
 * Pulp is introducing more data about container manifests to the API.
 This information allows Katello to display manifest labels, annotations, and information about the manifest type, such as if it is bootable or represents flatpak content.
 As a result, migrations must be performed to pull this content from manifests into the database.
 endif::[]
 
 ifdef::katello[]
++
```

Comment:
```suggestion
```

Reply:
Here it makes sense IMHO because the sentence refers to line 18.